pages/ltv/index.js

```diff
@@ -43,31 +36,12 @@ function LTV({ changeTheme }) {
   const onPoolSelectChange = (event, theOption) => {
     setAsset(theOption);
 
-    setLoading(true);
```

Comment:
This is indirectly unused, basically the `loading` state was no longer used (after some refactoring I suppose).
So it didn't make sense to keep the `setLoading()` call either.
So yeah I think it's better to take the dead code out and put it back later when we decide to use it again.
Same thing happen with some other functions that I took out since they were no longer used, e.g. `renderAssetOption()` below